mak/WINDOWS

```diff
@@ -1203,6 +1203,9 @@ $(IMPDIR)\core\sys\windows\rpcnterr.d : src\core\sys\windows\rpcnterr.d
 $(IMPDIR)\core\sys\windows\schannel.d : src\core\sys\windows\schannel.d
 	copy $** $@
 
+$(IMPDIR)\core\sys\windows\sdkddkver.d : src\core\sys\windows\sdkddkver.d
+	copy $** $@
+
```

Comment:
I believe you also need to add the file to `mak/COPY` and `mak/SRCS`. See the contribution guide, or where else existing files occur (e.g. the `secext.d` below).